cranelift/codegen/src/dominator_tree.rs

```diff
@@ -464,6 +464,20 @@ impl DominatorTreePreorder {
         }
     }
 
+    /// Checks if one instruction dominates another.
+    pub fn dominates_inst(&self, a: Inst, b: Inst, layout: &Layout) -> bool {
```

Comment:
This is a minor thing, but I would prefer to have `dominates_inst` next to the `dominates` method that it uses. Then someone looking at one of them can easily see the other one too.